heclib/heclib_c/src/Dss6/deleteFile.c

```diff
@@ -1,5 +1,5 @@
 #include <stdio.h>
-#ifdef __linux__
+#if defined __linux__ || defined __APPLE__
```

Comment:
if __UNIX__ may be better here.
with && not defined __SUNPRO_CC if solaris sunpro doesn't have unistd.

Reply:
Hi Mike, for this, I just tried with '__unix__' and it doesn't work for macOS, as it only defines 'APPLE' and 'MACH' https://stackoverflow.com/questions/142508/how-do-i-check-os-with-a-preprocessor-directive

Reply:
ah. that makes sense. arguably should still have the __unix__ and not just linux in there but that doesn't need to be part of your PR.

Reply:
Yup 😁 👌